docs/antctl.md

```diff
@@ -181,12 +183,25 @@ antrea-network-policy.md#antrea-native-policy-ordering-based-on-priorities).
 antctl get networkpolicy --sort-by=effectivePriority
 ```
 
-Antrea Agent additionally supports printing NetworkPolicies applied to a
-specified local Pod using this `antctl` command:
+Antrea Agent additionally supports some functions using this `antctl` command.
 
-```bash
-antctl get networkpolicy -p pod -n namespace
-```
+* Printing NetworkPolicies applied to a specified local Pod.
+
+  ```bash
+  antctl get networkpolicy -p pod -n namespace
+  ```
+
+* Printing NetworkPolicies with a specific source NetworkPolicy type: K8sNP, ACNP, ANP.
+  
+  ```bash
+  antctl get networkpolicy -T sourceType(K8sNP|ACNP|ANP)
+  ```
+
+* Printing NetworkPolicies with a specific name of its source NetworkPolicy.
```

Comment:
"Printing the NetworkPolicy of a specific source NetworkPolicy"

Reply:
I thought there could be several matches here if we don't specify the sourceType as well?

Reply:
Yes, different source NetworkPolicies may have the same name.

Reply:
Ok.